src/main/java/com/puppycrawl/tools/checkstyle/PackageObjectFactory.java

```diff
@@ -49,6 +49,20 @@
  * @author lkuehne
  */
 public class PackageObjectFactory implements ModuleFactory {
+
+    /**
+     * Enum class to define loading options.
+     */
+    public enum ModuleLoadOption {
+        /** Searching from packages given in countructor and from registred checkstyle modules. **/
+        SEARCH_REGISTERED_PACKAGES,
+        /**
+         * As SEARCH_REGISTERED_PACKAGES and also searching for class in checkstyle packages.
```

Comment:
@kazachka This item wasn't done. See https://github.com/checkstyle/checkstyle/pull/5245#discussion_r153035407 .

Reply:
updated, please check